src/index.ts

```diff
@@ -27,13 +29,37 @@ app.get("/me/movies", async (req: Request, res: Response) => {
   // """Vient""" du cookie
   const userId = '42';
 
-  const user = await UsersService.getById(userId)
+  // const user = await UsersService.getById(userId)
+  const user = await  UsersService.getAll();
+  console.log(user);
 
   // const userMovies = MoviesService.getByUser(userId);
 
   res.send(user);
 });
 
+//ajoute et affiche les utilisateur faut s'inspirer pour les movie etc
+app.get("/me/movie/create", async (req: Request, res: Response) => {
```

Comment:
Si cette route te sert à "créer des movies pour ton utilisateur", il faut respecter quelques normes :
- si c'est une création de données, utilises un `POST` et pas un `GET`
- quand tu mets un POST, c'est implicitement une création, tu peux retirer le /me/movie/~~create~~
- les entités que tu appelles dans ton API sont quasi toujours au pluriel `/me/movies`

```suggestion
app.post("/me/movies", async (req: Request, res: Response) => {
```